Documentation/cheatsheet.rst

```diff
@@ -2,157 +2,396 @@
+    cilium monitor --related-to=<id>
 
-	cilium monitor --type drop
 
+Filter for only events on layer 7
 ::
 
-	# Don't dissect packet payload, display payload in hex format
+    cilium monitor -t L7
 
-	cilium monitor -v --hex
-
-Endpoints
-=========
 
+Show notifications only for dropped packet events
 ::
 
-	# Get list of all local endpoints
+    cilium monitor --type drop
 
-	cilium endpoint list
 
+Don't dissect packet payload, display payload in hex information
 ::
 
-        # Get detailed view of endpoint properties and state
+    cilium monitor -v --hex
+
 
-        cilium endpoint get <id>
 
+Connectivity
+------------
+
+Check cluster Connectivity
 ::
 
-        # Show recent endpoint specific log entries
+	cilium-health status
 
-        cilium endpoint log <id>
+There is also a `blog post
+<https://cilium.io/blog/2018/2/6/cilium-troubleshooting-cluster-health-monitor>`_
+related with this tool.
+
+Endpoints
+---------
 
+Get list of all local endpoints
 ::
 
-	# Enable debugging output on the cilium monitor for this endpoint
+    cilium endpoint list
 
-	cilium endpoint config <id> Debug=true
+Get detailed view of endpoint properties and state
+::
 
-Tracing
-=======
+    cilium endpoint get <id>
 
+Show recent endpoint specific log entries
 ::
 
-	# Check policy enforcement between two labels on port 80
+    cilium endpoint log <id>
 
-	cilium policy trace -s <app.from> -d <app.to> --dport 80
+Enable debugging output on the cilium monitor for this endpoint
+::
+
+    cilium endpoint config <id> Debug=true
 
+
+Loadbalancing
+-------------
+
+Get list of loadbalancer services
 ::
 
-        # Check policy enforcement between two identities
+    cilium service list
+
+
+Or you can get the loadbalancer information using bpf list
+:::
 
-        cilium policy trace --src-identity <from-id> --dst-identity <to-id>
+    cilium bpf lb list
 
+
+Add a new loadbalancer
 ::
 
-        # Check policy enforcement between two pods
+    cilium service update --frontend 127.0.0.1:80 \
+        --backends 127.0.0.2:90,127.0.0.3:90 \
+        --id 20 \
+        --rev 2
 
-        cilium policy trace --src-k8s-pod <namespace>:<pod.from> --dst-k8s-pod <namespace>:<pod.to>
+BPF
+---
 
-Loadbalancing
-=============
+List node tunneling mapping information
+::
+
+    cilium bpf tunnel list
 
+Checking logs for verifier issue
+::
+
+    journalctl -u cilium | grep -B20 -F10 Verifier
 
+List connection tracking entries:
 ::
 
-        # Get list of loadbalancer services
+    sudo cilium bpf ct list global
 
-        cilium service list
+Flush connection tracking entries:
+::
 
-BPF
-===
+    sudo cilium bpf ct flush
 
+List proxy configuration:
 ::
 
-        # List node tunneling mapping information
+    sudo cilium bpf proxy list
+
+
+Kubernetes examples:
+=====================
+
+If you running Cilium on top of Kubernetes you may also want a way to list all
+cilium endpoints or policies from a single Kubectl commands. Cilium by using
```

Comment:
I would rephrase the sentence starting with `Cilium by using` to `Cilium provides all this information to the user by using (...)`